### DISCUSSION TITLE
[Help]: Set initial selectedScrollSnap

### DISCUSSION THREAD

**Discussion by zwelhtetyan** [davidjerleke/embla-carousel#817] [Help] [answered]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

For now, we can only rely on selectedScrollSnap and scrollSnapList. 

<img width="683" alt="Screenshot 2024-04-09 at 1 33 50 PM" src="https://github.com/davidjerleke/embla-carousel/assets/89287841/35c1f49c-744a-4ff1-869d-cd021cb5d9e7">

I my case, I want to set "15s" timer to default. 

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment]
@zwelhtetyan thanks for your feature request. However, I don't understand what you want to achieve.

**Reply by zwelhtetyan** [reply, marked as answer]
@davidjerleke Sorry for my bad description. I want slide start from index: 1. In my case, '15s'. I tried to change initial selectedScrollSnap. Now, I found an option `startIndex`. It works as expected. Anyway thanks for your reply ❤️‍🔥.

**Reply by davidjerleke** [reply]
@zwelhtetyan so [`startIndex`](https://www.embla-carousel.com/api/options/#startindex) solved your problem?

**Reply by zwelhtetyan** [reply]
@davidjerleke Yes. Now It works. `selectedScrollSnap` starts from 1. That is what I want.

<img width="517" alt="Screenshot 2024-04-09 at 1 51 50 PM" src="https://github.com/davidjerleke/embla-carousel/assets/89287841/81c3579c-cb0d-420b-b7c7-e0e8b9d27c99">
